crates/types/src/v1alpha1/wasmcloud_host_config.rs

```diff
@@ -165,6 +179,23 @@ fn pod_schema(_gen: &mut SchemaGenerator) -> Schema {
     val.schema.into()
 }
 
+/// This is a workaround for the fact that we can't override the Container schema to make name
+/// an optional field. It generates the OpenAPI schema for the Container type the same way that
+/// kube.rs does while dropping any required fields.
+fn container_schema(_gen: &mut SchemaGenerator) -> Schema {
+    let gen = schemars::gen::SchemaSettings::openapi3()
+        .with(|s| {
+            s.inline_subschemas = true;
+            s.meta_schema = None;
+        })
+        .with_visitor(kube::core::schema::StructuralSchemaRewriter)
+        .into_generator();
+    let mut val = gen.into_root_schema_for::<Container>();
+    // Drop `containers` as a required field, along with any others.
```

Comment:
Nit: remove this comment since it doesn't apply to this schema merge

Reply:
Updated the comment to reflect the appropriate field, thanks!